From b73e19eea357266ed74b3c74f8b3e1e85bd0bb9a Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Sat, 30 Dec 2023 08:44:16 +0100
Subject: [PATCH] com.webos.service.location: Remove gpsConfig file

Since we will provide this as part of nyx-conf, so we can make it device specific.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/etc/gpsConfig.conf | 2 --
 1 file changed, 2 deletions(-)
 delete mode 100644 files/etc/gpsConfig.conf

diff --git a/files/etc/gpsConfig.conf b/files/etc/gpsConfig.conf
deleted file mode 100644
index 637183b..0000000
--- a/files/etc/gpsConfig.conf
+++ /dev/null
@@ -1,2 +0,0 @@
-[GPSDEVICE]
-PORT=/dev/ttyUSB0
